src/slurm_plugin/clustermgtd.py

```diff
@@ -395,8 +396,8 @@ def manage_cluster(self):
             try:
                 log.info("Retrieving nodes info from the scheduler")
                 active_nodes, inactive_nodes = self._get_node_info_from_partition()
-                log.debug("Current active slurm nodes in scheduler: %s", active_nodes)
-                log.debug("Current inactive slurm nodes in scheduler: %s", inactive_nodes)
+                cloudwatch_metrics._slurm_active_nodes = active_nodes
```

Comment:
if `_slurm_active_nodes` is a private attribute (underscore prefix) we should not access to it.
We should have data encapsulation.

Let's use python properties: see for example this [guide](https://www.python-course.eu/python3_properties.php) or the [official documentation](https://docs.python.org/3/library/functions.html?highlight=property#property).

We're already using it somewhere in the CLI (e.g. for the region).

Reply:
At first it was a private attribute, forgot to change it back, thanks for the catch and the links.